Makefile.am

```diff
@@ -0,0 +1,30 @@
+## This is a automake file, part of the GPTL package.
+# Copyright 2018, see the COPYRIGHT file for more information.
+
+# This is the main automake file for GPTL.
+# Ed Hartnett, 5/18/18
+
+# This directory stores libtool macros, put there by aclocal.
+ACLOCAL_AMFLAGS = -I m4
+
+# These files get added to the distribution.
+EXTRA_DIST = COPYING INSTALL README
+
+# This is the list of subdirs for which Makefiles will be constructed
+# and run.
+SUBDIRS = include src
+
+install-data-hook:
+
+	@echo ''
+	@echo '+-------------------------------------------------------------+'
+	@echo '| Congratulations! You have successfully installed GPTL!      |'
+	@echo '|                                                             |'
+	@echo '| CAUTION:                                                    |'
+	@echo '|                                                             |'
+	@echo '| If you have not already run "make check", then we strongly  |'
+	@echo '| recommend you do so. It does not take very long.            |'
+	@echo '|                                                             |'
+	@echo '| GPTL is developed and maintained at NOAA.                   |'
```

Comment:
What should we say instead of "at NOAA" here. More detail would be useful.

Reply:
I would say just remove the "is developed" comment entirely, since I've been developing it through my last 5 jobs going all the way back to when I was at NCAR.

Reply:
It gives the package considerable street cred to cite support by prominent research institutions. How about something like:

GPTL was developed with support from multiple institutions, including NOAA and NCAR. It is maintained by the GSD group at NOAA.

Reply:
Yes that's better. Would be good to add ORNL to the lab list since I was there for a couple of years and worked on GPTL there also

Reply:
OK, done.